src/GUI/RevEng.Core/DbContextExtensions

```diff
@@ -12,8 +12,10 @@ namespace #NAMESPACE#
 {
     public static class DbContextExtensions
     {
-        public static async Task<T[]> SqlQueryAsync<T>(this DbContext db, string sql, IEnumerable<object> parameters, CancellationToken cancellationToken = default) where T : class
+        public static async Task<T[]> SqlQueryAsync<T>(this DbContext db, string sql, object[] parameters = null, CancellationToken cancellationToken = default) where T : class
         {
+            parameters ??= new object[] { };
```

Comment:
What is this syntax ? Will it work with C# 8 projects?

Reply:
> Available in C# 8.0 and later, the null-coalescing assignment operator ??= assigns the value of its right-hand operand to its left-hand operand only if the left-hand operand evaluates to null. The ??= operator doesn't evaluate its right-hand operand if the left-hand operand evaluates to non-null.

https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-coalescing-operator